Source/MediaInfo/File__Analyze_Streams.cpp

```diff
@@ -2507,7 +2507,8 @@ void File__Analyze::DisplayAspectRatio_Fill(const Ztring &Value, stream_t Stream
     else if (DAR>=(float)2.15 && DAR<(float)2.22)   DARS=__T("2.2:1");

```

Comment:
Sorry, looking at your commit i just saw that 2.2:1 line above and get suspicious. i couldn't resist to post a remark.
It look to me that it should be `2.21:1`

> +The following fixed values are supported:
> +@table @option
> +@item 4/3
> +@item 16/9
> +@item 221/100
> +@end table
> +Any other value will result in square pixels being signalled instead
> +(see H.262 section 6.3.3 and table 6-3).

Taken from [there](https://git.ffmpeg.org/gitweb/ffmpeg.git/blobdiff/2bc9ba8d3c41f3a8e56484bd67b05040c7909a01..b78c30d7ec26af67c00ce2002709a189f6a87a7e:/doc/bitstream_filters.texi).

Reply:
Is this potentially outside of the scope of the pull request/issue which was dealing with the 2.39:1 issue?

Reply:
Hehe. As i said sorry, i couldn't resist when i discovered it ;)
You are right, i should have open an other issue. I'll try to remember to do that later.

Reply:
:)